x/concentrated-liquidity/pool.go

```diff
@@ -100,7 +100,7 @@ func (k Keeper) CalcOutAmtGivenIn(ctx sdk.Context,
 	tokenOutDenom string,
 	swapFee sdk.Dec,
 	priceLimit sdk.Dec,
-	poolId uint64) (tokenIn, tokenOut sdk.Coin, updatedTick sdk.Int, updatedLiquidity sdk.Dec, err error) {
+	poolId uint64) (tokenIn, tokenOut sdk.Int, updatedTick sdk.Int, updatedLiquidity sdk.Dec, err error) {
```

Comment:
Extension from my previous Q: What's the reason for the change to sdk.Int?

IMO it should be sdk.Coin considering that existing CalcOutGiven in returns a coin and the point that the we would be expecting or constructing a coin any way in the caller of this method

Reply:
Yeah I think I agree with you, thinking back to why I changed this in the first place I think it was for troubleshooting purposes. Will change back